src/quo2/components/colors/color_picker/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.colors.color-picker.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [reagent.core :as reagent]
+            [quo2.components.colors.color-picker.style :as style]))
+
+(defn picker-colors
+  [blur?]
+  (concat (map (fn [color]
+                 {:name  color
+                  :color (colors/custom-color-by-theme color (if blur? 60 50) 60)})
+               [:blue :yellow :turquoise :copper :sky :camel :orange :army :pink :purple :magenta])
+          [{:name            :fengshui
```

Comment:
@Parveshdhull, should fengshui not just be added directly to the custom-color-by-theme function??

I thought that list is just the custom colors selected by design team so this should be alongside it 👍

Reply:
From my understand, `feng` and `chui` are two separate colors(see the [customization colors](https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Foundations?node-id=619%3A9895&t=J7cjdtt9EzPQIs5C-4)). The `custom-color-by-theme` is implemented in such a way that it can get variations of one color.

Here we are using the combination of these two colors to produce a unique picker element so that it will look good on both light & dark themes.

<img width="43" alt="Screenshot 2023-02-17 at 19 54 29" src="https://user-images.githubusercontent.com/19279756/219681152-ce901a7c-3092-46f2-8710-b5216336c336.png">

Reply:
Yea but as I understand it the function is designed to handle the custom Color's for users so we should adjust it to also handle two colors.
Let's get feedback from @Parveshdhull as initially implemented and knows it's use case

Reply:
yup we can't add this color, because its a component

how feng+chui = fengshui? (don't understand what they mean, just curious)